theorems/T000141.md

```diff
@@ -3,7 +3,7 @@ uid: T000141
 if:
```

Comment:
This is a duplicate of https://topology.pi-base.org/theorems/T000163 - better to keep the more direct result. (Also this erroneously refers to omega-Menger game.)

Reply:
Well, it was referring to omega-Menger before we collapsed the equivalence. Theorem 2.2 of the Combinatorics of Open Covers II says $\sigma$-compact $\implies \mathsf S_{\mathrm{fin}}(\Omega,\Omega)$, but the proof produces a Markov strategy. This is partly what I was referring to with the alias thing. Users will have to check properties for their aliases.

Reply:
I don't quite remember what the original incarnation of T141 was. I'm not sure it referred to Markov strategies. Should we simply remove this theorem?

Reply:
Oops this is a very old theorem, so for now I'll just mark as deprecated.